lib/usdUfe/ufe/trf/UsdTransform3dMatrixOp.cpp

```diff
@@ -41,14 +39,12 @@ PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace {
 
-using MayaUsd::ufe::UsdSetXformOpUndoableCommandBase;
-
-const char* getMatrixOp() { return std::getenv("MAYA_USD_MATRIX_XFORM_OP_NAME"); }
+using UsdUfe::UsdSetXformOpUndoableCommandBase;
 
 std::vector<UsdGeomXformOp>::const_iterator
 findMatrixOp(const std::vector<UsdGeomXformOp>& xformOps)
 {
-    auto opName = getMatrixOp();
+    auto opName = UsdUfe::getTransform3dMatrixOpName();
```

Comment:
This is where it now calls the DCC override function. In UsdUfe this returns nullptr, but in Maya it returns the value from the env var (if set) otherwise nullptr. The nullptr check is below on line 50.